### PR TITLE
Add eventing-XXX to repo config for sandbox

### PR DESCRIPTION
**Pull request by lionelvillard**
For 
- https://github.com/knative/community/issues/223
- https://github.com/knative/community/issues/224
- https://github.com/knative/community/issues/232
- https://github.com/knative/community/issues/233
- https://github.com/knative/community/issues/234
- https://github.com/knative/community/issues/235
- https://github.com/knative/community/issues/236
- https://github.com/knative/community/issues/237

@grantr as discussed.

### CODE REVIEW

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -386,8 +442,15 @@ orgs:
...
           discovery: admin
           downstream-test-go: admin
+          eventing-awssqs: admin
+          eventing-camel: admin
+          eventing-ceph: admin
+          eventing-couchdb: admin
+          eventing-github: admin
+          eventing-gitlab: admin
           eventing-kafka: admin
-          eventing-kafka-broker: admin
```
Comment (pierDipi) [inline peribolos/knative-sandbox.yaml:390]: Why?

**Review by pierDipi — CHANGES_REQUESTED** [review]

File: peribolos/knative-sandbox.yaml (hunk 1 above)
Comment (slinkydeveloper) [reply peribolos/knative-sandbox.yaml:390]: waaaat @lionelvillard ?

Comment (lionelvillard) [reply peribolos/knative-sandbox.yaml:390]: oops. Fixed. Thanks for catching this!

**Comment by lionelvillard** [comment]
/unhold

I fixed it.

**Comment by evankanderson** [comment]
/lgtm
/approve

Things seem to be alphabetical, the names match the tickets, and all the names are in all the places.

**Pull request merged**